mods/ra/maps/soviet-soldier-volkov-n-chitzkoi/rules.yaml

```diff
@@ -0,0 +1,204 @@
+World:
+	LuaScript:
+		Scripts: scu35ea.lua, scu35ea-AI.lua
+	MissionData:
+		Briefing: The Allies have stolen a new armor plating from us. With it, they can make structures nearly invulnerable. Their processing plant must be destroyed.\n\nFor this critical mission, we are allowing you access to Soldier Volkov. Use him and his cybernetic dog to sabotage the Allied control center, allowing our units in the southeast access to the alloy facility.\n\nThe mission is a success when Volkov destroys the control center, and our units destroy the alloy facility.\n
+		WinVideo: grvestne.vqa
+		LossVideo: sovcemet.vqa
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: easy
+
+Player:
+	LobbyPrerequisiteCheckbox@GLOBALBOUNTY:
+		Enabled: False
+		Locked: True
+
+MSLO:
+	Buildable:
+		Prerequisites: ~disabled
+        
+E7:
+	Buildable:
+		Prerequisites: ~disabled
+	Health:
+		HP: 25000
+	Armor:
+		Type: Heavy
+        
+2TNK:
+	Buildable:
+		Prerequisites: ~vehicles.allies, ~techlevel.medium
+        
+CA:
+	Buildable:
+		Prerequisites: ~syrd, ~techlevel.high
+        
+C2:
+	Inherits@2: ^ArmedCivilian
+        
+C3:
+	Inherits@2: ^ArmedCivilian
+        
+C4:
+	Inherits@2: ^ArmedCivilian
+        
+C5:
+	Inherits@2: ^ArmedCivilian
+        
+C6:
+	Inherits@2: ^ArmedCivilian
+        
+C8:
+	Inherits@2: ^ArmedCivilian
+        
+C9:
+	Inherits@2: ^ArmedCivilian
+        
+C10:
+	Inherits@2: ^ArmedCivilian
+        
+V01:
+	SpawnActorOnDeath:
+		Actor: healcrate
+        
+HOSP:
+	Tooltip:
+		GenericName: Hospital
+		GenericVisibility: Enemy, Ally, Neutral
+		GenericStancePrefix: false
+		TooltipDescription: Destroy this hospital to get healing equipment.
+	SpawnActorOnDeath:
+		Actor: healcrate
+        
+MONEYCRATE:
+	GiveCashCrateAction:
+		Amount: 2000
+
+^Vehicle:
+	-Demolishable:
+
+^Mine:
+	Tooltip:
+		ShowOwnerRow: false
+
+AFAC:
+	Inherits: BIO
+	Buildable:
+		Prerequisites: ~disabled
+	Health:
+		HP: 400000
+	Tooltip:
+		Name: Alloy Facility
+		GenericName: Alloy Facility
+		GenericVisibility: Enemy, Ally, Neutral
+		GenericStancePrefix: false
+		TooltipDescription: This facility produce our stolen armor plating.
```

Comment:
Same here.

Reply:
While you are at it, please change `produce` to `produces`.